.idea/compiler.xml

```diff
@@ -6,16 +6,16 @@
         <sourceOutputDir name="target/generated-sources/annotations" />
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
-        <module name="odysseusAT" />
+        <module name="AtlasTests" />
       </profile>
     </annotationProcessing>
-    <bytecodeTargetLevel>
-      <module name="odysseusAT" target="1.8" />
+    <bytecodeTargetLevel target="9">
```

Comment:
Why Java 9 in one place and 8th in another (line below)? Or do I get smth wrong?

Reply:
What I have to do with it?